Integrations/rasterize/rasterize.yml

```diff
@@ -16,6 +16,7 @@ configuration:
 description: Rasterize URLs or emails given to it
 display: Rasterize
 name: Rasterize
+fromversion: "5.0"
```

Comment:
please change to `fromversion: "5.0.0"`
this what prevents the build from running tests